From b6d4f884a402e766a5e40916e453626f74ed5aa0 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Thu, 4 Aug 2022 08:49:47 +0800
Subject: [PATCH] Revert "Linux 5.10.135"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 4fd9cb57a3f5e611efde7772643134385de3a5a6.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 5f4dbcb43307..00dddc2ac804 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 135
+SUBLEVEL = 134
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.34.1

